twiml/twiml.go

```diff
@@ -5,47 +5,34 @@ import (
 	"strings"
 )
 
-type Noun interface {
+type Tag interface {
```

Comment:
Think `Element` is a better name: https://www.twilio.com/docs/voice/twiml#twiml-elements